data/tlg0643/tlg001/tlg0643.tlg001.1st1K-mul2.xml

```diff
@@ -86,7 +86,7 @@
     <encodingDesc>
       <p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
       <refsDecl n="CTS">
-        <cRefPattern n="paragraph" matchPattern="(.+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:p[@n='$1'])"/>

```

Comment:
This was also correct. (See my comment about data/tlg0643/tlg001/tlg0643.tlg001.1st1K-mul1.xml .)
Also delete code line 108 please.